### DISCUSSION TITLE
What are the actual colors of Game Boy?

### DISCUSSION THREAD

**Discussion by P0W3** [visualboyadvance-m/visualboyadvance-m#1355] [General]
So, I am trying to figure out what are the actual colors of Game Boy.

I am trying to change from color to color, in order to get the actual colors of Game Boy.

To be accurate, which palette does Game Boy use?

Comparison of the Game Boy Pocket palettes.

This ([https://s5.postimg.org/lyq457avr/Option_1.png](url))
or
This ([https://s5.postimg.org/f89mvzl7r/Option_2.png](url))

Comparison of the Original Game Boy palettes.

This ([https://s5.postimg.org/74apifg1j/Option_3.png](url))
or
This ([https://s5.postimg.org/x14duw6sn/Option_4.png](url))

### Build details

VisualBoyAdvance-M (v2.0.2) (64-bit) (Windows)

**Comment by Squall-Leonhart** [comment]
grey scale, but the lcd on the dmg was green.
